EPRN.Portal/Services/WasteService.cs

```diff
@@ -404,5 +413,29 @@ public async Task<AccredidationLimitViewModel> GetAccredidationLimit(int journey
 
             return vm;
         }
+
+        public double? GetBaledWithWireDeductionPercentage(UserRole userRole)
+        {
+            if (userRole == UserRole.Exporter)
+                return ConfigSettings.Value.DeductionAmount_Exporter;
+            else if (userRole == UserRole.Reprocessor)
+                return ConfigSettings.Value.DeductionAmount_Reprocessor;
+            else if (userRole == UserRole.None)
+                return 0;
```

Comment:
I would return null here too - but we'll tackle that later. Let's just get this approved.